docs/src/development/ssh/troubleshoot-ssh.md

```diff
@@ -19,7 +19,7 @@ check it using `platform environment:list` or in the [management console](https:
 
 ## Redeploy your environment
 
-If you have just added your SSH key or made changes to [access rules](/administration/users.md), you need to redeploy your environment before you can access it using SSH keys. You can do this in the [management console](https://console.platform.sh/), by running `platform repdeploy`, or by pushing an empty git commit:
+If you have just added your SSH key or made changes to [access rules](/administration/users.md), you need to redeploy your environment before you can access it using SSH keys. You can do this in the [management console](https://console.platform.sh/), by running `platform redeploy`, or by pushing an empty git commit:
```

Comment:
Good catch. I forgot Vale ignores things in code blocks. There might be more things like this :see_no_evil: